java-surefire/src/main/java/org/sonar/plugins/surefire/SurefireSensor.java

```diff
@@ -56,13 +57,14 @@ public void describe(SensorDescriptor descriptor) {
 
   @Override
   public void execute(SensorContext context) {
-    File dir = SurefireUtils.getReportsDirectory(settings, fs, pathResolver);
-    collect(context, dir);
+    List<File> dirs = SurefireUtils.getReportsDirectories(settings, fs, pathResolver);
+    collect(context, dirs);
   }
 
-  protected void collect(SensorContext context, File reportsDir) {
-    LOGGER.info("parsing {}", reportsDir);
-    surefireJavaParser.collect(context, reportsDir, settings.hasKey(SurefireUtils.SUREFIRE_REPORTS_PATH_PROPERTY));
+  protected void collect(SensorContext context, List<File> reportsDirs) {
+    LOGGER.info("parsing {}", reportsDirs);
+    surefireJavaParser.collect(context, reportsDirs,
+      settings.hasKey(SurefireUtils.SUREFIRE_REPORT_PATHS_PROPERTY) || settings.hasKey(SurefireUtils.SUREFIRE_REPORTS_PATH_PROPERTY));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "SUREFIRE_REPORTS_PATH_PROPERTY"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)